Felienne/hedy
# [IMPROVEMENT] Updates last_login timestamp on each cookie retrieval

**Description**
Currently we only update our last_login record in the database on the moment of logging in and changing password. Due to the usage of session cookies to keep track of the current user these actions don't happen that often. Resulting in an incorrect last_login value that does not represent the users their activity. This can be annoying for teachers as this value is shown on the class overview. In the PR we implement an additional update moment. Each time a user is logged in from the cookie the timestamp is updated as well.

**Fix for**
This PR fixes #1790.

**How to test**
This is quite hard to test but this should work: Make sure you are a teacher and have another account as a student in your class. Notice the current last_login timestamp. Log out as a teacher, log in as the student to ensure there is a cookie for this user. Completely close your browser. When opening it again the student should still be logged in (from the cookie!). Log in as the teacher and verify that the last_login timestamp is updated to the moment of cookie retrieval and is not still the same value as the original login timestamp.

An easier solution to test: Make sure you are in your own class, close the browser and repeat the same steps. Your timestamp should be updated as well.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
